pkg/driver/efs_watch_dog.go

```diff
@@ -284,7 +285,11 @@ func (w *execWatchdog) updateConfig(efsClientSource string) error {
 	// used on Fargate, IMDS queries suffice otherwise
 	region := os.Getenv("AWS_DEFAULT_REGION")
 	fipsEnabled := os.Getenv("FIPS_ENABLED")
-	efsCfg := efsUtilsConfig{EfsClientSource: efsClientSource, Region: region, FipsEnabled: fipsEnabled}
+	portRangeUpperBound, found := os.LookupEnv("PORT_RANGE_UPPER_BOUND")
```

Comment:
You don't have to check the condition here, you can assign the portRangeUpperBound as you have already added it under env in the deployment helm charts.
`portRangeUpperBound := os. Getenv("PORT_RANGE_UPPER_BOUND")`

Also, one check should be added to ensure that portRangeUpperBound is always greater than portRangeLowerBound(20049).

Reply:
Let me know if there is something else or just waiting you resolve the conversation and provide the approval.

Reply:
Tagging @mskanth972 to help with approval.
Thanks!

Reply:
Hi all, when do you think it can be approved? Many thanks